Cargo.toml

```diff
@@ -73,6 +73,7 @@ exclude = [
 lazy_static = "1.4.0"
 libc = "0.2.93"
 bitflags = "1.2.1"
+num_cpus = "1.13.0"
```

Comment:
I suggest to skip this dependency and force users to pass the value themselves

Reply:
for the tests, I'll just put 2 as a default value?

Reply:
Sure, or maybe just use single-threaded version anyway?

Reply:
Ah, wait; don't you have to add the `--single-threaded` V8 flag everywhere then too?

Reply:
two is fine for the tests